.gitignore

```diff
@@ -2,3 +2,4 @@
 /bin/
 /vendor/
 composer.lock
+.phpunit.result.cache
```

Comment:
Can you please take this out of the file? Ideally, it will be part of your global `gitignore` settings, not part of the public repository.

Reply:
Done 👍

Reply:
Thank you 🍺